webpack/dev.config.js

```diff
@@ -19,6 +19,7 @@ var config = {
   devtool: 'eval-source-map',
   entry: {
     app: [
+      'babel-polyfill', // from YOU IE crashing fix
```

Comment:
cool took out YOU specific comment 9e7b2b7 and got id of the import in app/app.js 3846a67